cloudwatch_exporter_ecs.tf

```diff
@@ -0,0 +1,67 @@
+resource "aws_ecs_task_definition" "cloudwatch_exporter" {
+  family                   = "cloudwatch-exporter"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "512"
+  memory                   = "4096"
```

Comment:
aren't these set in `variables.tf`?

Reply:
These are the limits for the task not the amount used for the the amount used in the container. Unfortunately the can sometimes have to be increased based on the the number of sidecars.